Packages/vcs/Lib/projection.py

```diff
@@ -21,6 +21,9 @@
 round_projections = ['polar (non gctp)', 'stereographic',
                      'orthographic', "ortho", ]
 
+no_over_proj4_parameter_projections = round_projections+["aeqd", "lambert conformal c"]
```

Comment:
ok i see the issue, the parameter passed to the projection should not be `polar`, it should be `polar (non gctp)` or `-3`

The animation test are sol long I'm removing the polar projection from anim test. I don't think animating many projections is useful anyway, one test on an animated projection is enough.